test/functorch/test_ops.py

```diff
@@ -1197,6 +1197,7 @@ def get_vjp(cotangents, *primals):
         xfail('nn.functional.huber_loss', ''),  # NYI: forward AD for huber_loss_backward
         xfail('nn.functional.logsigmoid', ''),  # not differentiable w.r.t. buffer
         xfail('renorm', ''),  # NYI: forward AD for renorm
+        xfail('ormqr', ''),  # NYI: forward AD for ormqr
         xfail('symeig', ''),  # NYI: forward AD for symeig
```

Comment:
@kshitij12345 , @zou3519 , could you please help me out with `test_vmap` failures? `ormqr` is xfail whitelisted but still `test_vmap{jvp}/{vjp}` keep on failing.

Reply:
In functorch, each test has their own list op xfails and skips. The one highlighted is `test_jvpvjp`.

You will have to add relevant xfails for other tests.

Eg.
Here is the list for `test_vmapvjp`
https://github.com/pytorch/pytorch/blob/2c012c7109d9218e08533f859711098e489b7d58/test/functorch/test_ops.py#L710

Reply:
I see, I hoped there is a switch for all of them at once once the batching rules is not implemented...